staging/operator-registry/internal/action/render.go

```diff
@@ -322,6 +331,33 @@ func getRelatedImages(b *registry.Bundle) ([]declcfg.RelatedImage, error) {
 	if err = json.Unmarshal(*rawValue, &relatedImages); err != nil {
 		return nil, err
 	}
+
```

Comment:
Hi, @joelanford 
I use this branch to test "opm render" a bundle, but still cannot get operator images.
RC:
If there no spec.RelatedImages in csv, this function will return at line 327.

Reply:
Oh good catch! I'll make another upstream fix for that case.